grammar.js

```diff
@@ -197,6 +197,13 @@ module.exports = grammar({
     keyword_numeric: _ => make_keyword("numeric"),
     keyword_real: _ => choice(make_keyword("real"),make_keyword("float4")),
     keyword_float: _ => make_keyword("float"),
+    unsigned_integer: _ => seq(
```

Comment:
Would it be this type of issue that causes `$` to raise saying it's not found?

```suggestion
    unsigned_integer: $ => seq(
```